moveit_setup_assistant/templates/moveit_config_pkg_template/launch/moveit_rviz.launch

```diff
@@ -4,9 +4,9 @@
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
   <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
 
-  <arg name="config" default="false" />
-  <arg unless="$(arg config)" name="command_args" value="" />
-  <arg     if="$(arg config)" name="command_args" value="-d $(find [GENERATED_PACKAGE_NAME])/launch/moveit.rviz" />
+  <arg name="rviz_config" default="" />
+  <arg     if="$(eval rviz_config=='')" name="command_args" value="" />
```

Comment:
```suggestion
  <arg     if="$(eval rviz_config=='')" name="command_args" value="-d $(find [GENERATED_PACKAGE_NAME])/launch/moveit.rviz" />
```
We want to keep the default behavior

Reply:
I'd put the default two lines above the line you referenced: Imho it should still be possible, to set an empty config and pass nothing to rviz (which loads `~/.rviz/default.rviz`).

Reply:
The approach suggested in https://github.com/ros-planning/moveit/pull/1397#discussion_r267637419 should handle all cases, including the default:
* In `moveit_rviz.launch` the default of config=False would be kept. 
* `demo.launch` overrides this default by passing the package's `moveit.rviz`. 
* Launching `moveit_rviz.launch` with an explicit file argument for `config`, will load another rviz config. 

As far as I can see, you didn't exposed the `rviz_config` / `config` argument from `demo.launch`.

Reply:
@rhaschke @davetcoleman The current suggestion does not change default behaviour:
* The former default `False` did not pass any argument to rviz; same as the default `rviz_config==''` now.
* The default in demo.launch was to overwrite the argument with `True` to set the packages rviz-config which is the same now.

Reply:
Agreed. I think, @davetcoleman comment in https://github.com/ros-planning/moveit/pull/1397#discussion_r267583360, refers to not change the semantics of the variable "config", which was previously available to enable/disable the package's rviz config.

Reply:
ah i see, my bad.